scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpInfo.java

```diff
@@ -37,12 +38,28 @@ public interface OpInfo extends Comparable<OpInfo> {
 
 	/** Gets the op's input parameters. */
 	default List<Member<?>> inputs() {
-		return OpUtils.inputs(struct());
```

Comment:
Post-move, does anything still use `OpUtils.inputs` to obtain the members of a struct? Or can we slim down the `OpUtils` methods henceforth? It's not in an exported anymore, right? So we could trim it?